io.github.multiscrobbler.yml

```diff
@@ -0,0 +1,86 @@
+app-id: io.github.multiscrobbler
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: multiscrobbler
+finish-args:
+  - --share=network
+  # used for access to MPRIS in order to scrobble
+  - --own-name=org.mpris.MediaPlayer2.*
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+modules:
+  - name: multiscrobbler
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node18/bin
+      env:
+        XDG_CACHE_HOME: /run/build/multiscrobbler/flatpak-node/cache
+        npm_config_cache: /run/build/multiscrobbler/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node18
+        npm_config_offline: 'true'
+    build-commands:
+      # install npm dependencies
+      - npm install --offline
+
+      # build app
+      - npm run build
+
+      # makes node and npm available
+      - mkdir -p /app/bin /app/lib /app/lib/src
+      - cp -a /usr/lib/sdk/node18/bin/{node,npm} /app/bin
+      - cp -a /usr/lib/sdk/node18/lib/* /app/lib
+      - rm -r /app/lib/node_modules/npm/{docs,man}
+
+      # remove dev dependencies
+      - npm prune --production
+
+      # copy node_modules needed to run app
+      - cp -r node_modules/* /app/lib/node_modules
+
+      # delete typescript files (not needed for running app)
+      - find src/ -name "*.ts" -type f -delete
+
+      # copy app files to runtime dir
+      - cp -r src/* /app/lib/src
+      - cp -r config /app/lib/config
+      - cp ../defaultConfig.json /app/lib/config/config.json
+
+      - install -Dm644 ../${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ../icon.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 ../${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+
+      - install -Dm755 ../run.sh /app/bin/multiscrobbler
+    subdir: main
+    sources:
+      - type: git
+        url: https://github.com/FoxxMD/multi-scrobbler
+        branch: develop
```

Comment:
I currently have changes in `develop` branch for flatpak specifically but didn't want to tag a release until I was certain things were finalized on the flatpak/metainfo side so it would have an actual stable reference point. Can I just remove the branch or do i need to cut a release?

Reply:
you have a commit. which as soon as there is a new one on the branch will cause it fail because it doesn't. so *no branch* as I said.